zerver/webhooks/pingdom/doc.md

```diff
@@ -1,24 +1,33 @@
+# Zulip Pingdom integration
+
 Zulip supports integration with Pingdom and can notify you of
 uptime status changes from your Pingdom dashboard.
 
+{start_tabs}
+
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
+1. {!generate-webhook-url-basic.md!}
 
-1. Next, go to the following URL:
+1. In the main menu, go to **Integrations** under the **Integrations**
```

Comment:
Good improvement indeed.

```suggestion
1. From the main menu, open the "Integrations" menu, and
```
To make the order of the instructions same as the traversal steps, as well as conformity.